Sources/NetworkService/NetworkError/NetworkError+LocalizedError.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension NetworkError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .badRequest:
+            return "The server cannot process the request due to an error in the request."
+        case .authenticationError:
+            return "The server cannot complete the request because it lacks valid authentication credentials for the requested resource."
```

Comment:
The line is too long. Could you split that into two lines?

Reply:
Done.